src/client/pages/Path/Path.jsx

```diff
@@ -93,8 +93,8 @@ LessonCard.propTypes = {
   type: PropTypes.string.isRequired,
 };*/
 CourseCard.propTypes = {
-  pathId: PropTypes.objectOf(PropTypes.shape).isRequired,
-  courseId: PropTypes.objectOf(PropTypes.shape).isRequired,
+  pathId: PropTypes.string.isRequired,
+  courseId: PropTypes.string.isRequired,
   curriculum: PropTypes.objectOf(PropTypes.shape).isRequired,
```

Comment:
https://facebook.github.io/react/docs/typechecking-with-proptypes.html
yup shape is just `shape`, still, can't be used alone it requires the types for what's inside, can't be used alone